srcpkgs/wicd/INSTALL.msg

```diff
@@ -5,4 +5,5 @@ boot, please disable them before using wicd. You can do that with:
 	# rm -f /var/service/{dhcpcd,wpa_supplicant}
 
 ATTENTION: to make wicd work for your user, it must be a member
-of the 'users' group.
+of the 'network' group.You may need 'kdesu' for privilege 
```

Comment:
"group. You"

You're missing a space :)

Reply:
Fixed ;)